packages/hooks/src/useThrottle.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef, useState } from 'react';
+
+function useThrottle<T>(value: T, interval = 500): T {
```

Comment:
can we update it to a ES6 function.

Reply:
Add a documentation as well.

Reply:
@matin-deriv sure. and about the documentation you're absolutely right I forgot about it. will add it.